models/DataObject/AbstractObject.php

```diff
@@ -1359,7 +1359,7 @@ public static function __callStatic($method, $arguments)
         $db = \Pimcore\Db::get();
 
         if (in_array(strtolower($realPropertyName), self::$objectColumns)) {
-            $value = $arguments[0] ?? null;
+            $value = empty($arguments[0]) ? throw new \InvalidArgumentException('Mandatory argument $value not set.') : $arguments[0];
```

Comment:
Should be possible to write it like this, no? Or do you explicitly want to use `empty()` here?
```suggestion
            $value = $arguments[0] ?? throw new \InvalidArgumentException('Mandatory argument $value not set.');
```

Reply:
(only possible if the parameter must not be `null`)